src/library/styles/base/_spacing.scss

```diff
@@ -0,0 +1,17 @@
+.bsds-icon-right {
+  margin-left: space(md);
+}
+
+.bsds-icon-left {
+  margin-right: space(md);
+}
+
+.bsds-mt-6x {
+  margin-top: space(6x);
+}
+
+// TODO: Standardize utility names
```

Comment:
are we doing this now or later?
maybe in https://bsc-oit.atlassian.net/browse/ADS-631